core/jvm/src/main/scala/zio/sql/expr.scala

```diff
@@ -279,6 +280,9 @@ trait ExprModule extends NewtypesModule with FeaturesModule with OpsModule {
     val Ascii       = FunctionDef[String, Int](FunctionName("ascii"))
     val CharLength  = FunctionDef[String, Int](FunctionName("character_length"))
     val Concat      = FunctionDef[(String, String), String](FunctionName("concat"))
+    val ConcatWs2   = FunctionDef[(String, String), String](FunctionName("concat_ws"))
```

Comment:
So, `concat_ws` is not available at least for Oracle, so we need to do it in the db-specific modules. For now it's fine to do it in PostgresModule, although SQL Server also supports this function.

Reply:
Ok, in another PR